appeals/web/src/server/lib/mappers/appeal/submappers/s106-obligation-due-date.mapper.js

```diff
@@ -10,14 +10,14 @@ export const mapS106ObligationDue = ({ appealDetails, currentRoute }) => ({
 			},
 			value: {
 				// @ts-ignore
-				html: dateISOStringToDisplayDate(appealDetails.appealTimetable?.s106ObligationDue) || ''
+				html: dateISOStringToDisplayDate(appealDetails.appealTimetable?.s106ObligationDueDate) || ''
```

Comment:
are we able to remove the optional chaining now?

Reply:
mmm, what do you mean? It's just the field renamed, this wasn't in the DB

Reply:
removing the `?`

Reply:
I see, yes you're right

Reply:
ignore, timetable is null until the case is started...